.github/workflows/style.yml

```diff
@@ -29,10 +29,10 @@ jobs:
             ${{ runner.os }}-node-
 
       - name: Install dependencies
-        run: npm ci
+        run: npm install
```

Comment:
> This command is similar to [npm install](https://docs.npmjs.com/cli/v10/commands/npm-install), except it's meant to be used in automated environments such as test platforms, continuous integration, and deployment -- or any situation where you want to make sure you're doing a clean install of your dependencies.

Source: https://docs.npmjs.com/cli/v10/commands/npm-ci

Reply:
Got it